src/__tests__/lib/containers/oauth/OAuthManagement.test.tsx

```diff
@@ -0,0 +1,125 @@
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import moment from 'moment'
+import React from 'react'
+import { OAuthManagement } from '../../../../lib/containers/oauth/OAuthManagement'
+import { createWrapper } from '../../../../lib/testutils/TestingLibraryUtils'
+import { formatDate } from '../../../../lib/utils/functions/DateFormatter'
+import { useGetOAuthClientInfinite } from '../../../../lib/utils/hooks/SynapseAPI'
+import { server } from '../../../../mocks/msw/server'
+import {
+  mockUserProfileData,
+  mockUserProfileData2,
+} from '../../../../mocks/user/mock_user_profile'
+
+jest.mock('../../../../lib/utils/hooks/SynapseAPI/oauth/useOAuthClient', () => {
+  return {
+    useGetOAuthClientInfinite: jest.fn(),
+  }
+})
+
+const mockFetchNextPage = jest.fn()
+const mockGetOAuthClientInfinite = useGetOAuthClientInfinite as jest.Mock
+
+const mockClientList1 = {
+  results: [
+    {
+      client_id: mockUserProfileData.ownerId,
+      client_name: mockUserProfileData.firstName,
+      verified: false,
+      redirect_uris: [mockUserProfileData.url],
+      client_uri: mockUserProfileData.url,
+      policy_uri: mockUserProfileData.url,
+      tos_uri: mockUserProfileData.url,
+      createdOn: mockUserProfileData.createdOn,
+      modifiedOn: mockUserProfileData.createdOn,
+      createdBy: mockUserProfileData.userName,
+    },
+  ],
+  nextPageToken: '50a0',
+}
+
+const mockClientList2 = {
+  results: [
+    {
+      client_id: mockUserProfileData2.ownerId,
+      client_name: mockUserProfileData2.firstName,
+      verified: false,
```

Comment:
Reason for hardcoding verified=false?  I would think we would want one set to true and one false